torch/_subclasses/fake_tensor.py

```diff
@@ -479,6 +480,10 @@ def in_kernel_invocation_manager(fake_mode):
         del guard
 
 
+class FakeTensorConfig:
+    debug = os.environ.get("TORCH_FAKE_TENSOR_DEBUG", False)
```

Comment:
I hate how we're introducing another, different idiom for how to setup on the fly configs lol

Reply:
I dont love it either, lmk if you have any other place you prefer